### PR TITLE
Add `sourceOnly` option

### PR DESCRIPTION
**Pull request by rwwagner90**
This is used to essentially mark a list as immutable via dragging. It can only be used to copy items to other lists. The items cannot be rearranged or dragged out of the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -272,7 +273,36 @@ Incorrect:
...
+that you can only grab items from it to add to other buckets. The source bucket can never be
+reordered or modified by dragging items out of it. It is only a source to drag to other lists.
+
+This list would be marked as source only:
+
+```hbs
+{{#drag-sort-list
+  items         = items1
+  dragEndAction = (action 'dragEnd')
+  sourceOnly=true
```
Comment (lolmaus) [inline README.md]: Align assignment.

Comment (rwwagner90) [reply README.md]: Really not a fan of all these spaces, especially in hbs, but okay 👍

Comment (rwwagner90) [reply README.md]: Have you ever seen another Ember addon or app that does this? I have not.

**Comment by rwwagner90** [comment]
@lolmaus I just added a demo, and went to look at adding tests, but I am not familiar with the page object setup. I am somewhat surprised that adding the demo in the middle of the lists did not break other tests. Would you be able to help me write some tests please?

**Comment by rwwagner90** [comment]
@lolmaus any guidance for writing tests would be great!

**Comment by lolmaus** [comment]
@rwwagner90 I've created a follow-up PR: https://github.com/rwwagner90/ember-drag-sort/pull/6

**Comment by rwwagner90** [comment]
@lolmaus thanks so much for the help! I just merged your PR, so I think we should be good to go here?

**Comment by rwwagner90** [comment]
@lolmaus could we get this merged and a new beta out please? 😃

**Comment by rwwagner90** [comment]
@lolmaus any chance we could get this merged and get a new beta out?

**Comment by lolmaus** [comment]
@rwwagner90 Sorry for the delay, I'm on vacation. I'll try to have a look!

**Comment by lolmaus** [comment]
Released in `2.0.0-beta.2`, thx @rwwagner90 and sorry for the delay!

**Pull request merged**